packages/vscode-extension/src/controls/SampleGallery.tsx

```diff
@@ -60,6 +60,7 @@ const imageMapping: { [p: string]: any } = {
   "deep-linking-hello-world-tab-without-sso-M365": DeepLinking,
   "team-central-dashboard": DashboardTab,
   "hello-world-teams-tab-and-outlook-add-in": TeamsTabAndOutlookAddin,
+  "outlook-add-in-set-signature": TeamsTabAndOutlookAddin,
```

Comment:
```suggestion
  "outlook-add-in-set-signature": OutlookSetSignatureAddin,
```

The image to add to this PR is at https://github.com/OfficeDev/Office-Add-in-samples/raw/main/Samples/outlook-set-signature/assets/outlook-set-signature-overview.png. Can someone add this?
The import name is OutlookSetSignatureAddin if someone can add that around line 30 or so.